build.sh

```diff
@@ -42,7 +43,7 @@ MAKEFLAGS="--no-print-directory -j 9"
 
 export TARGET TARGET_FAMILY TARGET_SUBFAMILY TARGET_PROJECT PROJECT_PATH TOPDIR PREFIX_PROJECT PREFIX_BUILD\
 	PREFIX_BUILD_HOST PREFIX_FS PREFIX_BOOT PREFIX_PROG PREFIX_PROG_STRIPPED PREFIX_A\
-	PREFIX_H PREFIX_ROOTFS PREFIX_ROOTSKEL CROSS CFLAGS LDFLAGS CC LD\
+	PREFIX_H PREFIX_ROOTFS PREFIX_ROOTSKEL ROOTFS_OVERLAYS CROSS CFLAGS LDFLAGS CC LD\
```

Comment:
don't export it, no need to use/change it externally at build stage. Probably `PREFIX_ROOTSKEL` should also not be exported.

Reply:
Ok, you are right, I'll remove `PREFIX_ROOTSKEL` too.